src/python/WMCore/MicroService/Unified/Transferor.py

```diff
@@ -194,7 +194,11 @@ def __init__(self, config=None):
         self.config = config
         self.reqmgrAux = ReqMgrAux(self.config.reqmgr2_url)
         self.requests = {}
-        self.reqManager = RequestManager()
+        group = 'DataOps' if not hasattr(config, "group") else config.group
```

Comment:
How about
`group = getattr(config, "group", "DataOps")`
?
Same for the others.